openmdao_book/features/core_features/working_with_components/continuous_variables.ipynb

```diff
@@ -0,0 +1,924 @@
+{
```

Comment:
<p>are we going to automatically replace the asserts with prints like we do currently with sphinx?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMDAO/OpenMDAO_Book/pull/78/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>Since we're using the notebooks as the new place to "test" feature docs, we need to keep the asserts in. Also, because the code isn't a <code>unittest.TestCase</code>  we can't use things like <code>self.assertTrue</code>  so we have to convert some of them to simple asserts </p>